.github/workflows/scheduled-build.yml

```diff
@@ -2,8 +2,8 @@ name: release
 
 on:
   schedule:
-    # Runs "At minute 0 past every 2nd hour." (see https://crontab.guru)
-    - cron: "0 */2 * * *"
+    # Runs "At 12:00." (see https://crontab.guru)
```

Comment:
It looks like the release task is buggy due to the rust nightly compiler, we can reduce the frequency of this CI/CD task and wait for the bug to be officially fixed.

Reply:
done